package.json

```diff
@@ -8,8 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "gulp test",
-    "start": "gulp watch"
+    "test": "./node_modules/.bin/mocha test/*",
```

Comment:
That's quite another question and topic to discuss - `test` command can and will be used in development and not only on CI, that's for sure. I'd just leave it the way it was (i.e. `nyan` reporter when running `yarn test`) to avoid PR noise, as removing nyancat seems out of its scope here - WDYT?